plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -113,16 +127,21 @@
   force:
     description:
       - Allow to force stop VM.
-      - Can be used with states C(stopped) and C(restarted).
-    default: no
+      - Can be used with states C(stopped), C(restarted) and C(absent).
```

Comment:
I think `restarted` should not show up here.

Reply:
Is this module (`proxmox_kvm`) different in stable-1 ?

Reply:
https://github.com/ansible-collections/community.general/blob/29819e04ec30673b091eeaa89a7129b1da45a4d3/plugins/modules/cloud/misc/proxmox_kvm.py#L1163

It has an effect when it's been set with `restarted` state, what am I missing here?

Reply:
@felixfontein I think it is rather `absent` that should not be present here as it was considered a breaking change in #849.

Reply:
Sorry, I meant **`absent`** should not be there. (I.e. this line should not change in the diff.)

> Is this module (`proxmox_kvm`) different in stable-1 ?

Yes, there has been a breaking change that was not backported: https://github.com/ansible-collections/community.general/pull/849

Reply:
(The difference is also the reason why a manual backport is necessary.)

Reply:
Sure, thanks, fixed in https://github.com/ansible-collections/community.general/pull/1205/commits/bb67058c3d33cc7c7bf168cb247305201603ea5e.